### ISSUE TITLE
Docker stable version not found for the CentOS9

### ISSUE THREAD

**Issue by mrchiragrathod** [docker/for-linux#1336]
* [x] This is a bug report
* [ ] This is a feature request
* [ ] I searched existing issues before opening this one

### Expected behavior
Docker stable version should install on the CentOS9

### Actual behavior
Docker stable version not found for the installation

### Steps to reproduce the behavior
```
sudo yum install docker-ce docker-ce-cli containerd.io

```

**Output of `docker version:**

```
(Docker CE Stable - x86_64                       293  B/s | 383  B     00:01
Errors during downloading metadata for repository 'docker-ce-stable':
  - Status code: 404 for https://download.docker.com/linux/centos/9/x86_64/stable/repodata/repomd.xml (IP: 54.182.0.11)
Error: Failed to download metadata for repo 'docker-ce-stable': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried)

```

**Output of `docker info`:**

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.)**

**Comment by linghengqian** [comment]
Related to https://github.com/docker/roadmap/issues/303 , part of Roadmap.

**Comment by waynon** [comment]
I have the same issue.

**Comment by Romain-Geissler-1A** [comment]
Hi,

The just released release 20.10.15 is shipped with CentOS 9 packages, so this can be closed now.

Cheers,
Romain

**Comment by thaJeztah** [comment]
Yup! Let me close
